packages/cli/src/commands/serve.js

```diff
@@ -77,37 +79,14 @@ export const builder = async (yargs) => {
           )
           await bothSsrRscServerHandler(argv)
         } else {
-          // Wanted to use the new web-server package here, but can't because
-          // of backwards compatibility reasons. With `bothServerHandler` both
-          // the web side and the api side run on the same server with the same
-          // port. If we use a separate fe server and api server we can't run
-          // them on the same port, and so we lose backwards compatibility.
-          // TODO: Use @redwoodjs/web-server when we're ok with breaking
-          // backwards compatibility.
-          const { bothServerHandler } = await import('./serveBothHandler.js')
-          await bothServerHandler(argv)
```

Comment:
Nice to finally be able to get rid of this! <3

Reply:
@Tobbe hmmm maybe I should've left the comment—the behavior here hasn't changed. The route I've gone here is to try and minimize breaking changes so I'm using the server file as the way to get the new behavior (serving sides separately). I could make the change by default though if you think it's worth it, we can talk tomorrow

Reply:
Tobbe and I talked; we're going to make the change for the RC